config/jobs/kubernetes/sig-testing/integration.yaml

```diff
@@ -2,7 +2,7 @@ presubmits:
   kubernetes/kubernetes:
   # this replaces the bootstrap / scenario based job going forward
   - name: pull-kubernetes-integration
-    cluster: k8s-infra-prow-build
+    cluster: eks-prow-build-cluster
```

Comment:
The diff looks odd because it's not aligned with jobs.

The net effect is:
- move pull-kubernetes-integration from  k8s-infra-prow-build to eks-prow-build-cluster
- run it with settings of the now removed pull-kubernetes-integration-eks job